source/uri-options/tests/read-preference-options.json

```diff
@@ -0,0 +1,47 @@
+{
+  "tests": [
+    {
+      "description": "Valid read preference options are parsed correctly",
+      "uri": "mongodb://example.com/?readPreference=primaryPreferred&readPreferenceTags=dc:ny,rack:1&maxStalenessSeconds=120&readPreferenceTags=dc:ny",
+      "valid": true,
+      "warning": false,
+      "hosts": null,
+      "auth": null,
+      "options": {
+        "readPreference": "primaryPreferred",
+        "readPreferenceTags": [
+          "dc:ny,rack",
```

Comment:
This does not reflect the `rack:1` from the YAML file. You might need to regenerate the JSONs.

Reply:
Forgot to update the JSON files before pushing; I'll do that now